### PR TITLE
Add support for HTTP transport

### PR DESCRIPTION
**Pull request by jzding**

### CODE REVIEW

File: docs/development.md [hunk 1]
```diff
@@ -26,7 +25,10 @@ qdrouterd &
 ```shell
 cd <cloud-event-proxy repo>
 make build-plugins
-make run
+# Test with HTTP Transport
+go run cmd/main.go --transport-host="localhost:9043" --http-event-publishers="localhost:9043"
```
Comment (aneeshkp) [inline docs/development.md:29]: consumer and publisher exposing to same service ?

File: hw-event-proxy/go.mod [hunk 2]
```diff
@@ -4,7 +4,7 @@ go 1.17
 
 require (
 	github.com/json-iterator/go v1.1.12
-	github.com/redhat-cne/sdk-go v0.1.1-0.20220526200012-719244ead29f
+	github.com/redhat-cne/sdk-go v0.1.1-0.20220822193014-8b3ad5ad0a44
```
Comment (aneeshkp) [inline hw-event-proxy/go.mod:7]: we should try to make this point to latest every time it is build, that have we can reduce PR's. What do you think?

File: docs/development.md (hunk 1 above)
Comment (jzding) [reply docs/development.md:29]: yes. This is a local test so the consumer and publisher share the same side car service.

File: hw-event-proxy/go.mod (hunk 2 above)
Comment (jzding) [reply hw-event-proxy/go.mod:7]: we could use tags like v1.0.0

**Review by aneeshkp — APPROVED** [review]

**Pull request merged**